com.airtame.Client.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
What is the rationale for creating our own desktop file?

Reply:
I think the appstream check reported some issues a while back. I'll check again.

Reply:
Yea, if the desktop is placed in export/ it looks for the icons there but can't find any.